contrib/buildrpm/buildrpmLibfabric.sh

```diff
@@ -0,0 +1,362 @@
+#!/bin/bash
+
+# Copyright (c) 2016      Cisco Systems, Inc.  All rights reserved.
+#
+# Written by: Landon Curt Noll, Martin Kontsek
+#
+# This software is available to you under a choice of one of two
+# licenses.  You may choose to be licensed under the terms of the GNU
+# General Public License (GPL) Version 2, available from the file
+# COPYING in the main directory of this source tree, or the
+# BSD license below:
+#
+#     Redistribution and use in source and binary forms, with or
+#     without modification, are permitted provided that the following
+#     conditions are met:
+#
+#      - Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      - Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials
+#        provided with the distribution.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+# BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+# ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+####################################################################
+#                                                                  #
+#                  Libfabric RPM build script                      #
+#                                                                  #
+####################################################################
+
+####################
+# initialized values
+####################
+export specfile="libfabric.spec"
+export rpmbuilddir="$PWD/rpmbuild"
+export prefix="/usr"
+export noop=""
+export install_in_opt=""
+export create_modulefile=""
+export unpack_spec=""
+export verbose=""
+export st=""
+
+##################################
+# If you need custom configure and
+# rpmbuild options, put them here
+# or use "-c" and "-r" parameters
+##################################
+export configure_options=""
+export rpmbuild_options=""
+
+###########
+# functions
+###########
+verbose()
+{
+  if [[ -n "$verbose" ]]; then
+    echo "$0: --> $*"
+  fi
+}
+
+runcmd()
+{
+  verbose "Executing: $*"
+  if [[ -z "$noop" ]]; then
+    eval "$*"
+    st="$?"
+    if [[ "$st" != 0 ]]; then
```

Comment:
Super minor nits:
1. When assigning one shell variable to another, you don't need quotes.  I.e., it's *always* valid to do `foo=$bar` (even if `$bar` contains spaces).  This is doubly true because `$?` is guaranteed to be an integer.
1. It's odd style to compare a string to an integer (i.e., `"$st" != 0`).  My preference would be to compare integers (`$st -ne 0`), but you could do strings on both sides, too (`"$st" != "0"`).